src/Vision/VNUtils.cs

```diff
@@ -0,0 +1,74 @@
+﻿//
+// VNUtils.cs
+//
+// Authors:
+//	Alex Soto  <alexsoto@microsoft.com>
+//
+// Copyright 2017 Xamarin Inc. All rights reserved.
+//
+
+#if XAMCORE_2_0
+
+using System;
+using System.Runtime.InteropServices;
+using XamCore.CoreGraphics;
+using XamCore.ObjCRuntime;
+using XamCore.Foundation;
+
+using Vector2 = global::OpenTK.Vector2;
+
+namespace XamCore.Vision {
+	[TV (11,0), Mac (10,13, onlyOn64: true), iOS (11,0)]
+	public static partial class VNUtils {
+	
+		[Field ("VNNormalizedIdentityRect", Constants.VisionLibrary)]
+		public static CGRect NormalizedIdentityRect { get; } = Dlfcn.GetCGRect (Libraries.Vision.Handle, "VNNormalizedIdentityRect");
+
+		[DllImport (Constants.VisionLibrary)]
+		static extern bool VNNormalizedRectIsIdentityRect (CGRect rect);
+
+		public static bool IsIdentityRect (CGRect normalizedRect) => VNNormalizedRectIsIdentityRect (normalizedRect);
```

Comment:
Please merge the p/invoke with the managed caller to reduce the metadata. There's no point in having the both since there's no validation required in managed code.